02LessonTwo/homework/index.html

```diff
@@ -11,12 +11,15 @@
 
 <body bgcolor="#6ecff6">
 
-<h1 style="color:blue">Daniel Cisneros</h1>
+
+<h1 style="color: blue">Daniel Cisneros</h1>
 <hr> <!--Add hr is an important step-->
   <p style:"font-family: Times, serif; font-size: 12px;">1413 Dirt Lane, Austin Tx.</p>
 
```

Comment:
Thanks for the tips! I fixed it and hopefully I understood right. If not
then I definitely need a bit more help in lab.
On May 10, 2016 10:05 AM, "Jake Bapple" <notifications@github.com> wrote:

> In 02LessonTwo/homework/index.html
> <https://github.com/AustinCodingAcademy/intro-workbook/pull/25#discussion_r62689257>
> :
>
> > @@ -11,12 +11,15 @@
> >
> >  <body bgcolor="#6ecff6">
> >
> > -<h1 style="color:blue">Daniel Cisneros</h1>
> > +
> > +<h1 style="color: blue">Daniel Cisneros</h1>
> >  <hr> <!--Add hr is an important step-->
> >    <p style:"font-family: Times, serif; font-size: 12px;">1413 Dirt Lane, Austin Tx.</p>
> >
>
> be careful when indenting. the only times we indent is to indicate a
> parent>child relationship. in this case and in several more in the page,
> you have a paragraph indented like a child of an h2 or hr when they are
> only children of the div.
>
> Also, try adding in a section or main!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/AustinCodingAcademy/intro-workbook/pull/25/files/fd80dfad793f13672bb6877b7d601d7ce6fd4d20..6708298e907af2c460a0f86451fb5f6f955b0ac6#r62689257>
>